pipeline/build/stages.groovy

```diff
@@ -246,35 +246,36 @@ def buildIso(boolean cleanWorkspace = true) {
                           params.EPEL_ALTERNATE_MIRROR_RELEASE_URL)
     }
 
-    lock(resource: "build-iso_workspace_$env.NODE_NAME") {
-      echo 'Building ISO'
+    lock(resource: "build-images_workspace_$env.NODE_NAME") {
+      echo 'Building images'
       catchError {
         sh """\
 python host_os.py \\
        $params.HOST_OS_EXTRA_PARAMETERS \\
-       build-iso \\
+       build-images \\
+           --install-tree \\
            --packages-dir ../repository \\
            --iso-version $ISO_VERSION \\
-           $params.BUILD_ISO_EXTRA_PARAMETERS \\
+           $params.BUILD_IMAGES_EXTRA_PARAMETERS \\
 """
-        buildInfo.build_iso_finished = true
+        buildInfo.build_images_finished = true
       }
     }
   }
 
-  if (buildInfo.build_iso_finished) {
+  if (buildInfo.build_images_finished) {
     sh 'ln -s builds/result/iso/latest iso'
     stash name: 'iso_dir', includes: 'iso/'
```

Comment:
I think the stash name should be called "images_dir" to be consistent with the other names. Do you think the default result dir should also be changed, or "iso" is a more relatable name to our users?

Reply:
I'm renaming `iso_dir` to `images_dir` in stash calls.

As to `iso` being the resulting directory, it's fine, IMO. For example:

When only iso is built:

```iso/*.iso```

When only install tree is built:

```iso/os/```

When both iso and install tree are built:

```
iso/*.iso
iso/os/
```